site/pmguides/src/google-oauth-in-postman/google-oauth-in-postman.md

```diff
@@ -0,0 +1,192 @@
+author: Debo
+id: google-oauth-in-postman
+summary: Authorize your Google API's in Postman using OAuth 2.0
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/loopDelicious/pmquickstarts
+tags: Getting Started, Developer
+
+# Google OAuth in Postman
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
+
+Authentication and authorization are fundamental parts of working with APIs, and while there are different standards that define how both are done, the most popular and widely used standard for authorization is the [Open Authorization 2.0 standard](https://www.rfc-editor.org/rfc/rfc6749), referred to as OAuth 2.0 for short. 
+
+Large scale and enterprise organizations use the OAuth 2.0 as a standard way of authorizing access to their users data, and it has grown over the years to become an industry standard.
+
+The Postman API client let’s you work with different types of API authorization methods, including the OAuth 2.0. It makes the process of generating an authorization code and exchanging this code for an access token easy, and even provides you with an option that automatically refreshes your authorization tokens when they expire, given that a refresh token was returned from the authorization server.
+
+Millions of people across the globe use OAuth to authorize third party access to their data. In this tutorial, you will learn how you can use Postman to Authorize a Google API, and how this can be used in your API development and testing workflow. 
+
+### Prerequisites
+- Familiarity with OAuth concepts
+
+
+### What You’ll Learn   
+- How to create and Setup a new project on GCP
+- How to Authorize Google APIs using OAuth 2.0 on Postman
+
+### What You’ll Need 
+- A [GCP account](https://console.cloud.google.com/)
+
+
+<!-- ------------------------ -->
+## Create and setup a new Project
+Duration: 2
+
+Navigate to [console.cloud.google.com](https://console.cloud.google.com) and select the drop down at the top navigation menu. Click on the "New project" button.
+
+![New project](assets/new-project.png)
+
+Give your project a name, select a location and an organization.
+
+![Project Details](assets/project-details.png)
+
+The notification modal will pop up. Click the "Select Project" button fro the newly created project. 
```

Comment:
typo (fro -> for). I also would recommend using bold text when you reference UI elements instead of putting it inside quotes